catalyst/cypress/sandbox/catalyst-0-6-0/index.js

```diff
@@ -1,143 +1,153 @@
 setTimeout(() => {
-    const rule = window.evolv.catalyst['test-0-6-0'];
-    const store = rule.store;
-    const $ = rule.select;
-    const $$ = rule.selectAll;
-    const $i = rule.selectInstrument;
-    const { log, instrument } = rule;
-
-    instrument.add([
-        ['body', () => $('body'), { type: 'single' }],
-        ['navItem', () => $$('.nav-item')],
-        ['productNavItemLink', () => $$('.nav-item a[href="product.html"]')],
-        ['learnButton', () => $('#learn'), { type: 'single' }],
-        ['jumbotron', () => $('.jumbotron'), { type: 'single' }],
-        ['when-methods', () => $('.when-methods'), { type: 'single' }],
-    ]);
-
-    // select, selectAll, selectInstrument
-    $('h2').addClass('evolv-select-first-h2');
-    $$('//p').addClass('evolv-select-all-p');
-    $i('learnButton').addClass('evolv-select-single-instrument');
-    $i('navItem').addClass('evolv-select-all-instrument');
-
-    // rule.track
-    rule.track('c1v1');
-
-    // WHEN functions
-    // Here we can only test the active state because there is no live Evolv snippet and SPA routing setup
-    rule.whenContext('active').then(() => {
-        $('.when-methods .row').append(
-            `<div class="when-context col-md-4">
+  const rule = window.evolv.catalyst['test-0-6-0'];
+  const store = rule.store;
+  const $ = rule.select;
+  const $$ = rule.selectAll;
+  const $i = rule.selectInstrument;
+  const { log, instrument } = rule;
+
+  instrument.add([
+    ['body', () => $('body'), { type: 'single' }],
+    ['navItem', () => $$('.nav-item')],
+    ['productNavItemLink', () => $$('.nav-item a[href="product.html"]')],
+    ['learnButton', () => $('#learn'), { type: 'single' }],
+    ['jumbotron', () => $('.jumbotron'), { type: 'single' }],
+    ['when-methods', () => $('.when-methods'), { type: 'single' }],
+  ]);
+
+  // select, selectAll, selectInstrument
+  $('h2').addClass('evolv-select-first-h2');
+  $$('//p').addClass('evolv-select-all-p');
```

Comment:
Does this add a class to all 'p' elements?  Out of curiosity, why use xpath here?

Reply:
I wanted to have a test that used the XPath side of the select function